jettnet/src/Sockets/Steamworks/SteamManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using jettnet.core;
+using Steamworks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
You should edit your default VS template to not add so many unused namespaces

Reply:
Yeah good idea. My IDE scans namespaces for other types so I shouldn't need to add any to begin with... I'll change it now so I don't forget